### PR TITLE
providing an example on certificat auth with the api

### PR DESCRIPTION
**Pull request by sferich888**
#### What is this PR About?
Brief explanation of the documentation or playbook changes you have made and/or the new content you are contributing.

Adding example / guidance on certificate auth with the api. 

#### How should we test or review this PR?
Include any information or commands (please use code blocks) to help reviewers understand and verify your contribution.

Had to test this with case work for customer support issues. 

#### Is there a relevant Trello card or Github issue open for this?
Provide a link to any open Github issues or Trello cards that describe the use case or problem you are solving.

No

#### Who would you like to review this?
cc: @etsauer  can you review (you were cc'ed in an email with @liggit and @deads)

### CODE REVIEW

File: playbooks/operationalizing/integration.adoc [hunk 1]
```diff
@@ -59,6 +52,28 @@ All put together, a command line API request might look something like:
...
+
+* Identitiy certificate and key: `/etc/origin/master/admin.crt and /etc/origin/master/admin.key` 
+* Content-type Header: `Content-Type: application/json`
+* HTTP Method: `{GET|POST|PUT|PATCH|DELETE}`
+* API URL: `master.ose.example.com:8443`
+* Path ``/<api>/<version>/`
+** The Kubernetes API path is `/api/v1/`
+** The OpenShift API path is `/oapi/v1/`
+
+It should be note, that the *Authentication Header* is missing from the list above and replaced by a "certificate" and "key".
```
Comment (lpsantil) [inline playbooks/operationalizing/integration.adoc]: Minor typo

It should be noted,

File: playbooks/operationalizing/integration.adoc [hunk 2]
```diff
@@ -59,6 +52,28 @@ All put together, a command line API request might look something like:
...
+* Content-type Header: `Content-Type: application/json`
+* HTTP Method: `{GET|POST|PUT|PATCH|DELETE}`
+* API URL: `master.ose.example.com:8443`
+* Path ``/<api>/<version>/`
+** The Kubernetes API path is `/api/v1/`
+** The OpenShift API path is `/oapi/v1/`
+
+It should be note, that the *Authentication Header* is missing from the list above and replaced by a "certificate" and "key". 
+
+When building an API call in this form, its beast to use an https://docs.openshift.com/container-platform/latest/admin_guide/allocating_node_resources.html#system-resources-reported-by-node[example] that shows / highlights actual data, like that stats endpoint for a node.
```
Comment (lpsantil) [inline playbooks/operationalizing/integration.adoc]: Minor typo/grammar

When building an API call in this form, it's best to use an

Comment (etsauer) [reply playbooks/operationalizing/integration.adoc]: Also `s/like that/like the/`.

**Comment by etsauer** [comment]
@sferich888 some minor typo fixes, needed, then we'll be good to merge.

**Comment by sferich888** [comment]
Fixed the typos.

**Comment by lpsantil** [comment]
LGTM

**Pull request merged**